helm/linkerd2-multicluster-app/charts/partials/templates/_noop.tpl

```diff
@@ -0,0 +1,8 @@
+{{- define "partials.noop" -}}
+securityContext:
+  runAsUser: 65535
+args:
+- -v
+image: gcr.io/google_containers/pause:3.2
```

Comment:
I think this should also use the image registry switch mechanism with a giant swarm image

Reply:
yes, I noticed this but I want to rework it in a separated ticket because it affects all the apps. There are a few other images to fix. If you are ok with this, I'll fix them all for this ticket in another PR https://github.com/giantswarm/giantswarm/issues/24328